tests/integ/modin/test_apply_snowpark_python_functions.py

```diff
@@ -71,6 +71,7 @@ def test_apply_snowpark_python_function_not_implemented():
         pd.DataFrame({"a": [1, 2, 3]}).apply(asc, args=(1, 2))
 
 
+@pytest.mark.skip("SNOW-1859087 snowflake.cortex.summarize SSL error")
```

Comment:
Can we only skip it when it is run by jenkins?

Reply:
Yes, I added that now

Reply:
I updated the PR title accordingly.